kernel/src/main/scala/cats/kernel/Eq1.scala

```diff
@@ -0,0 +1,62 @@
+package cats.kernel
+
+/**
+ * Lifting of the [[Eq]] class to unary type constructors.
+ *
+ * @see [[https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Functor-Classes.html#t:Eq1]]
+ */
+trait Eq1[F[_]] extends Any with Serializable {
+
+  /**
+   * Lift equality tests through the type constructor.
+   */
+  def liftEq[A, B](compare: (A, B) => Boolean, x: F[A], y: F[B]): Boolean
```

Comment:
IMO, this name `liftEq` is not really apt here. This methods does not lift anything into some other context (as most of other `lift*` methods do in Cats) – it just does the comparison itself. Maybe it'd be worth to poke around the word `compare` instead, wdyt?

Reply:
I'm torn. I agree that the way we are using "lift" here is different than how we often use it. OTOH, I really like to keep names in line with any prior art and that is what `Eq1` uses in Haskell. https://hackage.haskell.org/package/base-4.14.1.0/docs/Data-Functor-Classes.html#v:liftEq

If the PR is otherwise approved, and the consensus is to use a different name, then I'd be happy to change it. My preference for using the canonical name from other language's implementations is only slight here.

If we do change it, `compare1` comes to mind as an option? Though I don't love digits in method names.

Reply:
Maybe `eqv1` ?